ote/src/clj/ote/integration/export/geojson.clj

```diff
@@ -59,41 +60,74 @@
-                                      {::t-operator/id transport-operator-id}))
-        service (first (specql/fetch db ::t-service/transport-service
-                                     transport-service-properties-columns
-                                     {::t-service/transport-operator-id transport-operator-id
-                                      ::t-service/id transport-service-id
-                                      ::t-service/published op/not-null?}))
-        service (link-to-companies-csv-url service)
-        operator-without-personal-info (dissoc operator ::t-operator/gsm ::t-operator/visiting-address ::t-operator/email ::t-operator/phone)
-        service-without-personal-info (dissoc service ::t-service/contact-address ::t-service/contact-email ::t-service/contact-phone)]
-    (if (and (seq areas) operator service)
+                 (fn [{:keys [geojson primary?]}]
+                   (assoc (cheshire/decode geojson keyword)
+                     :style {:fill (if primary? "green" "orange")}))
+                 areas)})
+
+(defn- append-nap-generated-netex-file-links
+  "NOTE: needs ::t-service/external-interface id property for external-interfaces in order to
+  copy `data-content` from interface into matching generated NeTEx link.
+  Returns `service` collection where external-interfaces is appended with interfaces with url to NAP NeTEx download link."
+  [service db {{base-url :base-url} :environment} transport-service-id]
+  (when service
+    (let [netex-conversions (fetch-conversions db transport-service-id)]
+      (update service
+              ::t-service/external-interfaces
+              #(vec
+                 (concat []
+                         %
+                         (for [conversion netex-conversions]
+                           {:format "NeTEx"
+                            :data-content (:ote.db.netex/data-content conversion)
+                            ::t-service/external-interface (export-netex/file-download-url base-url
+                                                                                           transport-service-id
+                                                                                           (:ote.db.netex/id conversion))})))))))
+
+(defn- export-geojson [db config transport-operator-id transport-service-id]
+  (let [areas (seq
+                (fetch-operation-area-for-service db
+                                                  {:transport-service-id transport-service-id}))
+        operator (when areas
+                   (-> (first
+                         (specql/fetch db
+                                       ::t-operator/transport-operator
+                                       transport-operator-properties-columns
+                                       {::t-operator/id transport-operator-id}))
+                       ;; Personal data should not be exported due to privacy requirement
```

Comment:
Not personal data. It is company data
So the comment could be something like
;; Due to privacy requirement remove gsm, phone, address and email from operator export